mig-frankfurt/dataelementhub.model
# #13 - Add namespace grant management

**What's in the PR**
- change DeHubUserPermission to contain userAuthId instead of username, firstname, lastname and email
- add method to get all userpermissions for a namespace (to be decided if anyone but admins can read this)
- fix bug in save user method (now correctly returns database id of newly created user)
- add method to create or delete user access from namespace
- fix bug where users could have more than one grant in namespace (change constraint in dehub-dal too!)

**How to test manually**
* see [dehub-rest #22](https://github.com/mig-frankfurt/dataelementhub.rest/pull/22)
